internal/services/orbital/contact_resource.go

```diff
@@ -0,0 +1,205 @@
+package orbital
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/orbital/2022-03-01/contact"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/orbital/2022-03-01/contactprofile"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/orbital/2022-03-01/spacecraft"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ContactResource struct{}
+
+type ContactResourceModel struct {
+	Name                 string `tfschema:"name"`
+	ResourceGroup        string `tfschema:"resource_group_name"`
+	Spacecraft           string `tfschema:"spacecraft"`
+	ReservationStartTime string `tfschema:"reservation_start_time"`
+	ReservationEndTime   string `tfschema:"reservation_end_time"`
+	GroundStationName    string `tfschema:"ground_station_name"`
+	ContactProfileId     string `tfschema:"contact_profile_id"`
+}
+
+func (r ContactResource) Arguments() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"resource_group_name": azure.SchemaResourceGroupName(),
+
+		"spacecraft": {
```

Comment:
is this the Spacecraft ID, if so, can we rename it that?

```suggestion
		"spacecraft_id": {
```

Reply:
Sure. Renamed.